ste/sender-blobFolders.go

```diff
@@ -46,9 +46,9 @@ func newBlobFolderSender(jptm IJobPartTransferMgr, destination string, p pipelin
 	}
 	fromTo := jptm.FromTo()
 	if fromTo.IsUpload() {
-		out = &dummyUploader{fsend}
+		out = &dummyFolderUploader{fsend}
 	} else {
-		out = &dummys2sCopier{fsend}
+		out = &dummyFolderS2SCopier{fsend}
```

Comment:
These changes are not related to symlink. Do we need them in this PR?

Reply:
I believe these changes came alongside the merge from dev between AzCopy versions.